cki-send.py

```diff
@@ -40,36 +40,54 @@ def main(args):
             for line in report_file.readlines():
                 test, status, _ = line.split()
```

Comment:
Can we add the duration to the objects? (in seconds) :angel:

Reply:
Ok if I add that and maybe maintainers in a separate MR?

Reply:
Definitely :)